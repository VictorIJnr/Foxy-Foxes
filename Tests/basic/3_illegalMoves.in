1
1
2
2
3
2
5
4
3
5
3
4
