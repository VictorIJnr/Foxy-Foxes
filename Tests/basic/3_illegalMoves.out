     o o o    
     o o o    
 o o o o o o o
 o o . . . o o
 . . . . . . .
     . * .    
     . . .    
Geese play. Enter move:
Illegal move!
     o o o    
     o o o    
 o o o o o o o
 o o . . . o o
 . . . . . . .
     . * .    
     . . .    
Geese play. Enter move:
Illegal move!
     o o o    
     o o o    
 o o o o o o o
 o o . . . o o
 . . . . . . .
     . * .    
     . . .    
Geese play. Enter move:
Illegal move!
     o o o    
     o o o    
 o o o o o o o
 o o . . . o o
 . . . . . . .
     . * .    
     . . .    
Geese play. Enter move:
End of input reached. Goodbye.
